oeps/oep-0022-bp-django-caches.rst

```diff
@@ -0,0 +1,229 @@
+===========================
+OEP-0022: Caching in Django
+===========================
+
++-----------------+--------------------------------------------------------+
+| OEP             | :doc:`OEP-0022 </oeps/oep-0022-bp-django-caches>`      |
++-----------------+--------------------------------------------------------+
+| Title           | Caching in Django                                      |
++-----------------+--------------------------------------------------------+
+| Last Modified   | 2018-05-21                                             |
++-----------------+--------------------------------------------------------+
+| Authors         | Robert Raposa <rraposa@edx.org>                        |
++-----------------+--------------------------------------------------------+
+| Arbiter         | Alex Dusenbery <adusenbery@edx.org>                    |
++-----------------+--------------------------------------------------------+
+| Status          | Under Review                                           |
++-----------------+--------------------------------------------------------+
+| Type            | Best Practice                                          |
++-----------------+--------------------------------------------------------+
+| Created         | 2018-05-04                                             |
++-----------------+--------------------------------------------------------+
+| `Review Period` | 2018-06-06                                             |
++-----------------+--------------------------------------------------------+
+
+Context
+-------
+
+We have a variety of tools for caching in Django in various repositories,
+but the solutions are not in a shared repository and don't meet all needs. A
+variety of issues were uncovered while working on caching fixes and
+improvements within one of our IDAs.
```

Comment:
"IDA" is not an industry standard term - you may want to expand it - or just say "microservice".

Reply:
Thanks.  I'll go with "services" if that works for you.